src/types/SemanticVersion.ts

```diff
@@ -0,0 +1,16 @@
+/**
+ * @license Copyright (c) 2003-2024, CKSource Holding sp. z o.o. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+export type SemanticVersion = `${ number }.${ number }.${ number }`;
+
+/**
+ * Checks if the given string is a semantic version.
+ *
+ * @param version - The string to check.
+ * @returns `true` if the string is a semantic version, `false` otherwise.
+ */
+export function isSemanticVersion( version: string | undefined | null ): version is SemanticVersion {
+	return !!version && /^\d+\.\d+\.\d+$/.test( version );
```

Comment:
This will fail when using `nightly`, `rc`, `alpha` and `beta` releases. Consider removing `$`.

Reply:
Good catch!

Reply:
Fixed